Polygon_mesh_processing/include/CGAL/Polygon_mesh_processing/mesh_smoothing.h

```diff
@@ -0,0 +1,346 @@
+// Copyright (c) 2018 GeometryFactory (France).
+// All rights reserved.
+//
+// This file is part of CGAL (www.cgal.org).
+// You can redistribute it and/or modify it under the terms of the GNU
+// General Public License as published by the Free Software Foundation,
+// either version 3 of the License, or (at your option) any later version.
+//
+// Licensees holding a valid commercial license may use this file in
+// accordance with the commercial license agreement provided with the software.
+//
+// This file is provided AS IS with NO WARRANTY OF ANY KIND, INCLUDING THE
+// WARRANTY OF DESIGN, MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE.
+//
+// $URL$
+// $Id$
+// SPDX-License-Identifier: GPL-3.0+
+//
+//
+// Author(s)     : Konstantinos Katrioplas (konst.katrioplas@gmail.com)
+
+#ifndef CGAL_POLYGON_MESH_PROCESSING_MESH_SMOOTHING_H
+#define CGAL_POLYGON_MESH_PROCESSING_MESH_SMOOTHING_H
```

Comment:
mesh_smooth.h and shape_smooth.h ?
Or are you talking about merging to one?
And who introduced the -ing in PMP in the first place anyway?? :)

Reply:
Then it would become `smooth_mesh()` and `smooth_shape()`?
It's more consistent with the rest of the package indeed.
@sloriot should we go for that?

I did introduce `isotropic_remeshing()` but it's a mistake. And the header is `remesh.h` anyway...

Reply:
Let's go for
- `PMP::smooth_mesh()` in `smooth_mesh.h`, and
- `PMP::smooth_shape()` in `smooth_shape.h`
then!

Reply:
https://cgal.geometryfactory.com/CGAL/Members/wiki/Features/Small_Features/Smoothing#Summary_of_API_changes

I changed the header names.